### ISSUE TITLE
Malevolent Machine Learning

### ISSUE THREAD

**Issue by CrispyValorousQuarry** [CrispyValorousQuarry/CapstoneResearchPage#4]
[Malevolent Machine Learning](https://dl.acm.org/doi/pdf/10.1145/3365573) 
### Summary/Key Points
- In this article, research on deep learning is shown. It discusses the difficulties of AI vs human brain image recognition. The AI will notice and be thrown off by small things that human brains ignore. Generative adversarial networks were created to rebuild source images without the high-frequency noise (things that distract the computer from classifying an image properly, such as a sticker on a sign, glasses on a face). 
- Adversarial training provided the deep neural networks with a set of examples during the training phase to try to force the model to ignore miscellaneous features that have shown to throw off the software. This technique showed that this training can hurt the performance of the DNN on normal test images due to overfitting.

> “When training our model with adversarial training, we explicitly discourage it from relying on non-robust features. Thus, we are forcing it to ignore information in the input that would be useful for classification…one could argue, however, that the loss in accuracy is not necessarily a bad thing.”
